daemon/graphdriver/overlay/overlay.go

```diff
@@ -263,6 +300,20 @@ func (d *Driver) Create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 		}
 	}()
 
+	driver := &Driver{}
+	if opts != nil {
+		if err := d.parseStorageOpt(opts.StorageOpt, driver); err != nil {
+			return err
+		}
+	}
+
+	if driver.options.quota.Size > 0 && !strings.HasSuffix(dir, "-init") {
+		// enforce container disk project quota
+		if err := d.quotaCtl.SetQuota(dir, driver.options.quota); err != nil {
```

Comment:
I tried it but I remember that if I set a quota to upper directory, container's storage is not limited by quota. (and when 'df' in container, it doesn't show limited size)
I'll check it again and write a result soon.